### PR TITLE
PXB-2881 Rework the side TOC

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   mkdocs-base.yml

### CODE REVIEW

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]
Added my comments on GD

File: mkdocs-base.yml [hunk 1]
```diff
@@ -114,103 +125,154 @@ extra:
...
+            - xbcloud/xbcloud_azure.md
+            - xbcloud/xbcloud_minio.md
+      - xbcloud/xbcloud_exbackoff.md
+      - advanced/log_enhancements.md
+      - advanced/locks.md
+      - advanced/smart_memory_estimation.md
+      - xtrabackup_bin/flush-tables-with-read-lock.md
+      - xtrabackup_bin/working_with_binary_logs.md
+  -  Quick start guide:
+      - quickstart.md
```
Comment (patrickbirch) [inline mkdocs-base.yml]: Doesn't appear in TOC

File: mkdocs-base.yml [hunk 2]
```diff
@@ -114,103 +125,154 @@ extra:
...
+        - Run Percona XtraBackup: installation/yum-run-xtrabackup.md
+        - Uninstall: installation/yum-uninstall-xtrabackup.md
+      - Use binary tarballs:
+        - Install with binary tarballs: installation/binary-tarball.md
+        - Binary tarballs available: installation/binary-tarball-names.md
+      - Compile from source:
+        - installation/compiling_xtrabackup.md
+      - Docker:
+        - installation/docker.md
+      - Secure a backup:
```
Comment (patrickbirch) [inline mkdocs-base.yml]: Change to 
Secure
These files do not secure a backup - they secure an installation

File: mkdocs-base.yml [hunk 3]
```diff
@@ -114,103 +125,154 @@ extra:
...
+        - Back up individual partitions:
+          - howtos/recipes_xbk_partition.md
+          - Create an individual partitions backup: create-an-individual-partitions-backup.md
+          - Prepare an individual partitions backup: prepare-an-individual-partitions-backup.md
+        - Backup features:
+          - xtrabackup_bin/lru_dump.md
+          - advanced/throttling_backups.md
+        - Encrypt a backup:
+          - xtrabackup_bin/backup.encrypting.md
+          - advanced/encrypted_innodb_tablespace_backups.md
```
Comment (patrickbirch) [inline mkdocs-base.yml]: This should be moved to Features

File: mkdocs-base.yml [hunk 4]
```diff
@@ -114,103 +125,154 @@ extra:
...
+          - Prepare a compressed backup: prepare-a-compressed-backup.md
+        - Partial backup:
+          - xtrabackup_bin/partial_backups.md
+          - Create a partial backup: create-a-partial-backup.md
+          - Prepare a partial backup: prepare-a-partial-backup.md
+        - Back up individual partitions:
+          - howtos/recipes_xbk_partition.md
+          - Create an individual partitions backup: create-an-individual-partitions-backup.md
+          - Prepare an individual partitions backup: prepare-an-individual-partitions-backup.md
+        - Backup features:
```
Comment (patrickbirch) [inline mkdocs-base.yml]: Move these to Features

File: mkdocs-base.yml [hunk 5]
```diff
@@ -114,103 +125,154 @@ extra:
...
+        - Encrypt a backup:
+          - xtrabackup_bin/backup.encrypting.md
+          - advanced/encrypted_innodb_tablespace_backups.md
+      - Restore a backup:
+        - Restore types:
+          - Restore a full backup: restore-a-full-backup.md
+          - Restore an incremental backup: restore-an-incremental-backup.md
+          - Restore a compressed backup: restore-a-compressed-backup.md
+          - Restore a partial backup: restore-a-partial-backup.md
+          - Restore an individual partitions backup: restore-individual-partitions-backup.md
```
Comment (patrickbirch) [inline mkdocs-base.yml]: Is there a Restore an Encrypted backup topic?

File: mkdocs-base.yml [hunk 6]
```diff
@@ -114,103 +125,154 @@ extra:
...
+        - Restore features:
```
Comment (patrickbirch) [inline mkdocs-base.yml]: Move these to Features

Comment (alina-derkach-oaza) [reply mkdocs-base.yml]: moved

File: mkdocs-base.yml (hunk 5 above)
Comment (alina-derkach-oaza) [reply mkdocs-base.yml]: We have this info but that info is a part of another doc. I will add it explicitly.

File: mkdocs-base.yml (hunk 4 above)
Comment (alina-derkach-oaza) [reply mkdocs-base.yml]: Moved

File: mkdocs-base.yml (hunk 3 above)
Comment (alina-derkach-oaza) [reply mkdocs-base.yml]: moved

File: mkdocs-base.yml (hunk 2 above)
Comment (alina-derkach-oaza) [reply mkdocs-base.yml]: Updated

**Review by patrickbirch — APPROVED** [review]
Latest version is good

File: mkdocs-base.yml (hunk 1 above)
Comment (alina-derkach-oaza) [reply mkdocs-base.yml]: Added

**Comment by alina-derkach-oaza** [comment]
The changes are merged with https://github.com/percona/pxb-docs/pull/132

**Pull request closed without merging**